extensions/cornerstone/src/Viewport/OHIFCornerstoneViewport.tsx

```diff
@@ -345,6 +350,27 @@ const OHIFCornerstoneViewport = React.memo(props => {
     };
   }, [displaySets, elementRef, viewportId]);
 
+  // Set up the window level action menu in the viewport action corners.
+  useEffect(() => {
+    const wlActionMenu = getWindowLevelActionMenu({
+      viewportId,
+      viewportElem: elementRef.current,
+      displaySets,
+      servicesManager,
+      commandsManager,
+      verticalDirection: AllInOneMenu.VerticalDirection.BottomToTop,
+      horizontalDirection: AllInOneMenu.HorizontalDirection.LeftToRight,
```

Comment:
These can be down inside component

Reply:
Oh, hmmm I see these are something else. What are they actually?

Reply:
And they have to be here because they need to be set depending on which corner the component is added in.